automation/run-tests.sh

```diff
@@ -364,3 +364,4 @@ if [ $TEST_TYPE != $TEST_TYPE_ALL ] && \
     install_nmstate
 fi
 run_tests
+make license_header_check 
```

Comment:
`automation/run-tests.sh` does not run all the tests always, it allows to select what tests to run. The same way, you don't have to run the licenses check always. Study how run-tests.sh works and put this line in an appropriate place. Maybe we can run it when `$TEST_TYPE == $TEST_TYPE_FORMAT`.